### ISSUE TITLE
Missing outputs

### ISSUE THREAD

**Issue by kevinkoo001** [B2R2-org/FunSeeker#1]
Thanks for releasing an interesting tool for recognizing function boundaries.  With the Docker script, I was able to compile and run `FunSeeker` immediately with a provided sample. However, I do not see any output after the tool ends to run. Could you pinpoint what I might be doing something wrong? The followings are some of my attempts after mounting benchmark programs to `/benchmark`.

```
root@978a8653f354:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump
root@978a8653f354:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump type5
root@978a8653f354:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump fp
root@978a8653f354:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump endbr
root@978a8653f354:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump jump
```

**Comment by witbring** [comment]
Thank you for reporting. 
I have tried to figure out a way to solve your problem but I currently cannot reproduce the problem. 
I ran FunSeeker with our [dataset](https://drive.google.com/file/d/1tHv-Mws-rxy_3ErjSR6VS-0-xAKd0R0C/view?usp=sharing) and confirmed that FunSeeker properly works.
```
root@f0cc936e9e38:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ./FunSeeker /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump
51f20
c91c0
53720
...
```
Could you check a project version (commit number) and a test binary?
```
root@f0cc936e9e38:/FunSeeker/src/FunSeeker/bin/Release/net6.0# sha512sum /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump
0424ead573fae40b975adb5d90fd790255970a06cf843cd211efaccf5292490faab9e2cf93341b3b202dfa5a0d44c373ad974edb60f04d30a1fcee96f29e8c21  /benchmark/binutils-2.37/x64/clang/pie/ofast-lazy-bfd/bin/objdump
```

**Comment by kevinkoo001** [comment]
The last commit was 5f9cd8696b7b3ca651ad9bce44fdacd6d7d46a13. And I am using the dataset you have released. When I tried to run the first command as you did, it outputs nothing. Not sure what went wrong. Here is the directory I have.

```
root@0e49ae5d2008:/FunSeeker/src/FunSeeker/bin/Release/net6.0# ls -l
total 9416
-rwxr--r-- 1 root root  195584 Mar 17 14:55 B2R2.BinIR.dll
-rwxr--r-- 1 root root  522240 Mar 17 14:55 B2R2.Core.dll
-rwxr--r-- 1 root root  818176 Mar 17 14:55 B2R2.FrontEnd.BinFile.dll
-rwxr--r-- 1 root root   58368 Mar 17 14:55 B2R2.FrontEnd.BinInterface.dll
-rwxr--r-- 1 root root  985088 Mar 17 14:55 B2R2.FrontEnd.BinLifter.ARM32.dll
-rwxr--r-- 1 root root  667136 Mar 17 14:55 B2R2.FrontEnd.BinLifter.ARM64.dll
-rwxr--r-- 1 root root  136704 Mar 17 14:55 B2R2.FrontEnd.BinLifter.AVR.dll
-rwxr--r-- 1 root root   29184 Mar 17 14:55 B2R2.FrontEnd.BinLifter.CIL.dll
-rwxr--r-- 1 root root   53248 Mar 17 14:55 B2R2.FrontEnd.BinLifter.Core.dll
-rwxr--r-- 1 root root  140800 Mar 17 14:55 B2R2.FrontEnd.BinLifter.EVM.dll
-rwxr--r-- 1 root root 1469440 Mar 17 14:55 B2R2.FrontEnd.BinLifter.Intel.dll
-rwxr--r-- 1 root root  232448 Mar 17 14:55 B2R2.FrontEnd.BinLifter.MIPS.dll
-rwxr--r-- 1 root root   31744 Mar 17 14:55 B2R2.FrontEnd.BinLifter.Optimizer.dll
-rwxr--r-- 1 root root  158208 Mar 17 14:55 B2R2.FrontEnd.BinLifter.PPC32.dll
-rwxr--r-- 1 root root  194560 Mar 17 14:55 B2R2.FrontEnd.BinLifter.SH4.dll
-rwxr--r-- 1 root root  224256 Mar 17 14:55 B2R2.FrontEnd.BinLifter.TMS320C6000.dll
-rwxr--r-- 1 root root  241152 Feb  1  2020 FParsec.dll
-rwxr--r-- 1 root root  104448 Feb  1  2020 FParsecCS.dll
-rwxr--r-- 1 root root 3080328 May 24 14:49 FSharp.Core.dll
-rwxr-xr-x 1 root root  142632 Jul 22 05:30 FunSeeker
-rw-r--r-- 1 root root   17756 Jul 22 05:30 FunSeeker.deps.json
-rw-r--r-- 1 root root   35328 Jul 22 05:30 FunSeeker.dll
-rw-r--r-- 1 root root    6712 Jul 22 05:30 FunSeeker.pdb
-rw-r--r-- 1 root root     242 Jul 22 05:30 FunSeeker.runtimeconfig.json
drwxr-xr-x 2 root root    4096 Jul 22 05:30 cs
drwxr-xr-x 2 root root    4096 Jul 22 05:30 de
drwxr-xr-x 2 root root    4096 Jul 22 05:30 es
drwxr-xr-x 2 root root    4096 Jul 22 05:30 fr
drwxr-xr-x 2 root root    4096 Jul 22 05:30 it
drwxr-xr-x 2 root root    4096 Jul 22 05:30 ja
drwxr-xr-x 2 root root    4096 Jul 22 05:30 ko
drwxr-xr-x 2 root root    4096 Jul 22 05:30 pl
drwxr-xr-x 2 root root    4096 Jul 22 05:30 pt-BR
drwxr-xr-x 2 root root    4096 Jul 22 05:30 ru
drwxr-xr-x 2 root root    4096 Jul 22 05:30 tr
drwxr-xr-x 2 root root    4096 Jul 22 05:30 zh-Hans
drwxr-xr-x 2 root root    4096 Jul 22 05:30 zh-Hant
```

**Comment by witbring** [comment]
Hmm.. this is really weird. 
Could you show me your build log?
 ```
root@49d30e076f5d:/FunSeeker# dotnet build -c Release
Microsoft (R) Build Engine version 17.2.0+41abc5629 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
  FunSeeker -> /FunSeeker/src/FunSeeker/bin/Release/net6.0/FunSeeker.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:01.65
```

**Comment by witbring** [comment]
I think a file attribute might raise the problem since FunSeeker checks file attribute and works only when the file has no special attribute.
Thus, we decide to remove the check routine.

**Comment by witbring** [comment]
Fixed in [b15c338](https://github.com/B2R2-org/FunSeeker/commit/b15c338e6ef64b413d81c716a6027e8a8294b939)
I hope it will solve your problem.

**Comment by kevinkoo001** [comment]
The fix has resolved the issue. Thanks!!
